classes/migration/install/OJSMigration.php

```diff
@@ -318,14 +324,14 @@ public function up(): void
             $table->foreign('journal_id', 'subscription_types_journal_id')->references('journal_id')->on('journals')->onDelete('cascade');
             $table->index(['journal_id'], 'subscription_types_journal_id');
 
-            $table->float('cost', 8, 2);
+            $table->float('cost', 53);
```

Comment:
The `cost` column should definitely be converted over to a `decimal` now that we're losing the precision and scale.

Reply:
Right . In that case, we should have a migration also, added .